ngraph/core/include/ngraph/util.hpp

```diff
@@ -379,6 +379,8 @@ std::vector<T> read_vector(std::shared_ptr<ngraph::runtime::Tensor> tv)
     return rc;
 }
 
+std::vector<uint8_t> NGRAPH_API read_raw_data(std::shared_ptr<ngraph::runtime::Tensor> tv);
```

Comment:
As I see we use this method only in tests. So I propose to add this method only to tests.